util/include/util/odbtransaction.h

```diff
@@ -175,12 +175,22 @@ void persistAll(Cont& cont_, std::shared_ptr<odb::database> db_)
         << item->toString();
       LOG(warning)
         << ex.what() << std::endl
-        << "AST nodes in this translation unit will be ignored!";
+        << "Further changes in this transaction will might be ignored!";
```

Comment:
"will might be" -> "will be"

Reply:
I was not sure whether it is true for SQLite also, but will reword it then.

Reply:
Fixed in d0099ddf824925de4247256b084ca474631f6fc9.